google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/JsonStreamWriterTest.java

```diff
@@ -198,6 +199,14 @@ public void testTwoParamNewBuilder()
     assertEquals(TEST_STREAM, writer.getStreamName());
   }
 
+  @Test
+  public void testTwoParamNewBuilderDefault()
+      throws DescriptorValidationException, IOException, InterruptedException {
+    JsonStreamWriter writer =
```

Comment:
where is the two reflected?

Reply:
I see. Maybe name it as constructWriterUsingDefaultStreamName?

Reply:
Done